{
    "affected": [
        {
            "package": {
                "ecosystem": "Alpaquita:23",
                "name": "py3-urllib3",
                "purl": "pkg:apk/alpaquita/py3-urllib3?arch=source&distro=23"
            },
            "ranges": [
                {
                    "events": [
                        {
                            "introduced": "1.26.12-r1"
                        }
                    ],
                    "type": "ECOSYSTEM"
                }
            ]
        },
        {
            "package": {
                "ecosystem": "Alpaquita:stream",
                "name": "py3-urllib3",
                "purl": "pkg:apk/alpaquita/py3-urllib3?arch=source&distro=stream"
            },
            "ranges": [
                {
                    "events": [
                        {
                            "introduced": "1.26.12-r1"
                        }
                    ],
                    "type": "ECOSYSTEM"
                }
            ]
        }
    ],
    "id": "BELL-CVE-2025-50181",
    "modified": "2025-07-17T10:14:29.874414Z",
    "published": "2025-06-20T06:00:01.989358Z",
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://docs.bell-sw.com/security/cves/CVE-2025-50181"
        }
    ],
    "schema_version": "1.6.7",
    "severity": [
        {
            "score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N",
            "type": "CVSS_V3"
        }
    ],
    "upstream": [
        "CVE-2025-50181"
    ]
}
